script/local-ci.bat

```diff
@@ -1,6 +0,0 @@
-@echo off
-rem Run all the tests and linters in the CI pipeline locally
-pytest tests
-pylint src tests
-black --check src tests
-isort --check-only src tests
```

Comment:
The local-ci scripts are updated and can be kept. They run locally all the tests and linting checks that the GitHub actions use, allowing the developer to check locally if something is destined to fail.

Reply:
I restored the local-ci scripts, thanks